docs/section23/Rule23-8.md

```diff
@@ -0,0 +1,50 @@
+
+# Airside System - Rule 23-8  
+
+**Rule ID:** 23-8  
+**Rule Description:** For baseline systems 6 and 8, the minimum volume setpoint shall equal to 30% of peak primary design airflow or the rate required for minimum outside air whichever is larger.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**Appendix G Section Reference:** Section G3.1.3.14 Fan Power and Control (Systems 6 and 8)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-6 or 8  
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:**  
+
+1. baseline_hvac_type()  
+
+## Rule Logic:  
+
+- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
+
+  - For each terminal in zone: `for terminal_b in zone_b.terminals:`
+
+    - Get HVAC system serving terminal: `hvac_b = terminal_b.served_by_heating_ventilation_air_conditioning_systems`
+  
+      - Check if HVAC system is type 6 or 8: `if baseline_hvac_type(hvac_b) in ["SYS-6", "SYS-8"]:`
+
```

Comment:
I wonder if in this and other RDSs that depend on baseline system type we need to parse each zone and then each terminal and not just do for each building segment in each building get HeatingVentilationAirConditioningSystem. This seems to be more efficient, but perhaps it doesn't matter?